diff -Naur a/include/mupdf/fitz/config.h b/include/mupdf/fitz/config.h
--- a/include/mupdf/fitz/config.h	2019-05-28 01:27:38.335844989 +0100
+++ b/include/mupdf/fitz/config.h	2019-05-28 01:42:49.496523931 +0100
@@ -24,19 +24,19 @@
 	By default all but GPRF are enabled. To avoid building unwanted
 	ones, define FZ_ENABLE_... to 0.
 */
-/* #define FZ_ENABLE_PDF 1 */
-/* #define FZ_ENABLE_XPS 1 */
-/* #define FZ_ENABLE_SVG 1 */
-/* #define FZ_ENABLE_CBZ 1 */
-/* #define FZ_ENABLE_IMG 1 */
-/* #define FZ_ENABLE_HTML 1 */
-/* #define FZ_ENABLE_EPUB 1 */
-/* #define FZ_ENABLE_GPRF 1 */
+#define FZ_ENABLE_PDF 1
+#define FZ_ENABLE_XPS 0
+#define FZ_ENABLE_SVG 0
+#define FZ_ENABLE_CBZ 0
+#define FZ_ENABLE_IMG 0
+#define FZ_ENABLE_HTML 0
+#define FZ_ENABLE_EPUB 0
+#define FZ_ENABLE_GPRF 0
 
 /*
 	Choose whether to enable ICC color profiles.
 */
-/* #define FZ_ENABLE_ICC 1 */
+#define FZ_ENABLE_ICC 0
 
 /*
 	Choose whether to enable JPEG2000 decoding.
@@ -50,7 +50,7 @@
 	Choose whether to enable JavaScript.
 	By default JavaScript is enabled both for mutool and PDF interactivity.
 */
-/* #define FZ_ENABLE_JS 1 */
+#define FZ_ENABLE_JS 0
 
 /*
 	Choose which fonts to include.
@@ -61,31 +61,31 @@
 	unwanted fonts.
 */
 /* To avoid all noto fonts except CJK, enable: */
-/* #define TOFU */
+#define TOFU
 
 /* To skip the CJK font, enable: (this implicitly enables TOFU_CJK_EXT and TOFU_CJK_LANG) */
-/* #define TOFU_CJK */
+#define TOFU_CJK
 
 /* To skip CJK Extension A, enable: (this implicitly enables TOFU_CJK_LANG) */
-/* #define TOFU_CJK_EXT */
+#define TOFU_CJK_EXT
 
 /* To skip CJK language specific fonts, enable: */
-/* #define TOFU_CJK_LANG */
+#define TOFU_CJK_LANG
 
 /* To skip the Emoji font, enable: */
-/* #define TOFU_EMOJI */
+#define TOFU_EMOJI
 
 /* To skip the ancient/historic scripts, enable: */
-/* #define TOFU_HISTORIC */
+#define TOFU_HISTORIC
 
 /* To skip the symbol font, enable: */
-/* #define TOFU_SYMBOL */
+#define TOFU_SYMBOL
 
 /* To skip the SIL fonts, enable: */
-/* #define TOFU_SIL */
+#define TOFU_SIL
 
 /* To skip the Base14 fonts, enable: */
-/* #define TOFU_BASE14 */
+#define TOFU_BASE14
 /* (You probably really don't want to do that except for measurement purposes!) */
 
 /* ---------- DO NOT EDIT ANYTHING UNDER THIS LINE ---------- */
